bitwarden/server
# Update File Size to allow null

## Type of change

## Objective

Asana Ticket: https://app.asana.com/0/0/1201753604759430/f

Mobile sends over the file object with all `null` values. This is different from web which doesn't send over that property so Json was reading it as the default of `long` zero.  Making it a nullable `long` means mobile continues to work as before and it reflects the true value of what web sends `null`.

This will be 🍒⛏️ to `rc`.

## Code changes



* **SendFileModel.cs:** Made Size property nullable
* **SendService.cs:** Default to zero when Size value is needed as a non-nullable number

## Testing requirements

- Verify that you are able to create/edit/save Sends in:
  - web
  - mobile
  - desktop
  - browser
  - cli
- Verify that file sends count against user storage limits.
- Verify that a bad actor cannot grab the blob storage link and upload a file larger than the users storage allows. i.e they only have 1GB of storage but upload 2GB file.


## Before you submit
